### ISSUE TITLE
Named exports can break global

### ISSUE THREAD

**Issue by GMartigny** [purtuga/esm-webpack-plugin#12]
Hi,

When exporting named variable using the same name as a global scope variable, the two collide because of the way the plugin declare them.
This issue maybe related to the way webpack treat exports, but I'm not skilled enough on webpack build to tell for sure.

Here's an example:
```js
// math.js
export const square = x => Math.pow(x, 2);
```
```js
// index.js
import * as Math from "./math.js";

export {
    Math,
};
```
HTML file that consume the ESM build:
```html
<script type="module">
import { Math as M } from "./dist/build.esm.js";

console.assert(M.square(9) === 81);
</script>
```
`math.js:2 Uncaught TypeError: Math.pow is not a function`

Of course, this is a contrived example, but considering the number of variables under `window` in the browser scope, this can lead to many problems.
One solution could be to mangle the name of declared variables and export using `as`:

```js
// ./dist/build.esm.js.js
// Webpack build ...
const sdfhjfdbsdf = LIB['Math']
export {
    sdfhjfdbsdf as Math,
};
```

**Comment by purtuga** [comment]
Hi Martigny.
Thanks for making me aware of this issue. I will look at it in more detail
soon and consider your proposed solution - which is one I think will work
nicely.

Quick question: is your goal to really export a module member that has the
same name as a global?

/Paul
-- 
Paul T. -- Sent from Mobile

**Comment by GMartigny** [comment]
> is your goal to really export a module member that has the
same name as a global?

Yes, basically doing the same as my example. I have a [module named `Math`](https://github.com/pencil-js/pencil.js/tree/master/modules/math) holding some maths utilities. I also have an [`Image` module](https://github.com/pencil-js/pencil.js/tree/master/modules/image) which could trigger the same issue.

**Comment by purtuga** [comment]
Hi @GMartigny 
Sorry for the delay in addressing this issue. I just pushed a changed and also published a new version to NPM (1.2.1).
Thanks again for raising this issue and suggesting a fix :)

**Comment by GMartigny** [comment]
The fix [solve the issue](https://github.com/pencil-js/pencil.js/commit/692b9a6e190a257e97313e6ec0e0a19ae685134a).
Thanks a lot for you work.
👍
